demos/JWST/S4_template.ecf

```diff
@@ -31,12 +31,13 @@ fill_value      mask    # Either the string 'mask' to mask the outlier values (r
 
 # Limb-darkening parameters needed to compute exotic-ld
 compute_ld      True
-inst_filter     Prism #filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
-metallicity     0.1 #metallicity of the star 
-teff            6000 #effective temperature of the star in K
-logg            4.0 #surface gravity in log g
-exotic_ld_direc /home/User/exotic-ld_data/ #directory for ancillary files for exotic-ld
-exotic_ld_grid  3D #1D or 3D model grid
+inst_filter     prism   # Filter of JWST/HST instrument, supported list see https://exotic-ld.readthedocs.io/en/latest/views/supported_instruments.html 
+metallicity     0.1     # Metallicity of the star 
+teff            6000    # Effective temperature of the star in K
+logg            4.0     # Surface gravity in log g
+exotic_ld_direc /home/User/exotic-ld_data/ # Directory for ancillary files for exotic-ld, download from: https://zenodo.org/record/6344946#.Yt66YnaZND8
```

Comment:
Can you just point to https://zenodo.org/record/6344946 instead so we don't have to update this every time they release a new dataset?